### ISSUE TITLE
pip install django-parsley replaces Django 1.8.3 with 1.8

### ISSUE THREAD

**Issue by aptwebapps** [agiliq/Django-parsley#72]
I can't have Django 1.8.3 if I have django-parsley in my requirements file. Is this as intended? If not, perhaps changing

```
install_requires=["Django>=1.3,<=1.8"]
```

to

```
install_requires=["Django>=1.3,<1.9"]
```

or

```
install_requires=["Django>=1.3,<=1.8.6"]  # 1.8.6 is the latest 1.8 as of now
```

in setup.py would work? As it is, I can't have it my requirements file which makes automated deployments annoying. There are some import security updates in 1.8.2 and 1.8.3.

Actually, I _can_ have it in the requirements file if I move django-parsley to the top, although Django 1.8 is unnecessarily installed first.

**Comment by grafa** [comment]
+1 for this

**Comment by jeruyyap** [comment]
I just ran into this as well.

**Comment by k1-hedayati** [comment]
Please fix this issue

**Comment by busla** [comment]
+1.

I don´t think force-uninstalling the latest Django version to install django-parsley is a good idea.

**Comment by zachmccormick** [comment]
+1 on this as well - why would you ever force a lower version like this?!?  Put it in your docs that 1.8 is the highest officially supported version but never force it!!!

**Comment by jeruyyap** [comment]
Not sure if something changed with pip, but at least in my case this problem does not occur when installing via a requirements file. 
(even if I do not move it to the top as mentioned above)

Still, this really should be corrected properly.

**Comment by kmmbvnr** [comment]
This issue blocks pip-tools to proper resolve dependencies.

**Comment by NathHorrigan** [comment]
This still happens, FIX!!

**Comment by akshar-raaj** [comment]
@jproffitt This should be fixed with the recent 0.7 release, right?

**Comment by jproffitt** [comment]
@akshar-raaj yes, this is resolved in 0.7.
